node/hubController.js

```diff
@@ -289,6 +329,16 @@ class HubController {
     /**
      * helper methods
      */
+
+    /**
+     * Returns an array of strings, expected input is string:
+     *  "flight1,flight2"
+     */
+    _getFlights(flightInfo) {
+        return (flightInfo === undefined || flightInfo === null) ? [] :
+            flightInfo.toString().split(",").filter(n => n.length > 0);
```

Comment:
>n => n.length [](start = 52, length = 13)

unless you validate you cannot guarantee n is not null or undefined. check again here

Reply:
toString() converts null or undefined elements to empty strings, n cannot be null or undefined.